slack-api-model/src/main/java/com/slack/api/model/block/element/RichTextQuoteElement.java

```diff
@@ -19,5 +19,6 @@ public class RichTextQuoteElement extends BlockElement implements RichTextElemen
     private final String type = TYPE;
     @Builder.Default
     private List<RichTextElement> elements = new ArrayList<>();
+    private Integer border;
```

Comment:
Great catch; to make the automated tests pass, you need to run `./scripts/run_no_prep_tests.sh` to generate corresponding JSON files, which are used by other projects.

Reply:
Thanks for the guide! Will proceed.

Reply:
Resolved in 3afa2dcd02e0dbaaf4420f68c09870f303dc9eb9